src/components/button.js

```diff
@@ -41,6 +41,6 @@ const ButtonText = styled.Text `
 Button.propTypes = {
     children: PropTypes.element.isRequired,
     backgroundColor: PropTypes.string.isRequired
-};
+  };
 
 export default Button;
```

Comment:
Add an empty line at the end of the file (helps the stream input reader to know when the file read is over)